x/evm/types/utils.go

```diff
@@ -46,3 +52,24 @@ func rlpHash(x interface{}) (hash ethcmn.Hash) {
 
 	return hash
 }
+
+// GenerateReturnData takes all of the necessary data from the EVM execution
```

Comment:
Small change, correct comment with correct function name

Reply:
ah, fixed that in a different branch, will do